src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form>
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
```

Comment:
Apply code styling that is mentioned in checklist
![Screenshot 2023-03-21 at 18 20 19](https://user-images.githubusercontent.com/6526149/226673317-30216b6f-e2a1-49e7-b4e9-020f2ca721bd.png)

Reply:
Code styling applied in commit 6e85f76